infrastructure/modules/serverless/cloudtrail.tf

```diff
@@ -126,14 +109,18 @@ resource "aws_kms_key" "this" {
 EOF
 }
 
+data "aws_kms_alias" "existing" {
+  name = "alias/${var.namespace}-kms-key"
+}
 
 resource "aws_kms_alias" "a" {
+  count = var.create_kms && data.aws_kms_alias.existing.arn == null ? 1 : 0
 
-  count = var.create_kms ? 1 : 0
-
-  name          = "alias/${var.namespace}-kms-key"
-  target_key_id = aws_kms_key.this[0].key_id
+  name          = var.create_kms ? "alias/${var.namespace}-kms-key" : null
+  target_key_id = var.create_kms ? aws_kms_key.this[0].key_id : null
 }
 
 
 
+
+
```

Comment:
```suggestion
```

Reply:
Added in 621f2fb80404ba20e621b516b17cd84b11fbf1bb